_includes/02-image.md

```diff
@@ -0,0 +1 @@
+Replace this with an image, like your photo. Ensure you include some alt-text.
```

Comment:
![Picture of Jaime](https://user-images.githubusercontent.com/109662648/181019559-deb91bfc-7e51-4fda-86e5-0c2a181bc18d.jpg)